jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/AssetFileGenerator.java

```diff
@@ -689,54 +740,62 @@ public void generateFiles(List<SubscriptionVH> subscriptions, String filePath) {
 					return;
 				}
 				try {
-
+					targetTypeList.add("subscription");
 					FileManager
 							.generateSubscriptionFiles(subscriptionInventoryCollector.fetchSubscriptions(subscription));
 					log.info("subscription data saved!");
+					targetTypeList.remove("subscription");
 				} catch (Exception e) {
 					e.printStackTrace();
 					Util.eCount.getAndIncrement();
 				}
 			});
+
 			executor.execute(() -> {
 				if (!(isTypeInScope("functionapp"))) {
 					log.info("no target type found for functionApp!!");
 					return;
 				}
 				try {
+					targetTypeList.add("functionapp");
 					FileManager
 							.generateFunctionAppFiles(functionAppInventoryCollector.fetchFunctionAppDetails(subscription));
 					log.info("subscription data saved!");
+					targetTypeList.remove("functionapp");
 				} catch (Exception e) {
 					e.printStackTrace();
 					Util.eCount.getAndIncrement();
 				}
 			});
+
 			executor.execute(() -> {
 				if (!(isTypeInScope("mysqlflexible"))) {
 					log.info("no target type found for mysqlflexible!!");
 					return;
 				}
 				try {
-
+					targetTypeList.add("mysqlflexible");
 					FileManager
 							.generateMySQLFlexibleFiles(mySQLFlexibleInventoryCollector.fetchMySQLFlexibleServerDetails(subscription));
 					log.info("subscription data saved!");
```

Comment:
What is the point of these logs "subscription data saved"? I think we would benefit if you update these logs to match the blocks target type.

Also, I see that some executors above do not have log statements. For logging consistency pick if we have such logs, then update all executors blocks try/catch blocks to either have a log or not.

Reply:
okay @plyubich ...that is present from before. i will make it consistent.

Reply:
done.